bagelbids/pheno_utils.py

```diff
@@ -113,7 +113,9 @@ def get_age_heuristic(column: str, data_dict: dict) -> str:
     return data_dict[column]["Annotations"]["Transformation"]["TermURL"]
 
 
-def transform_age(value: Union[int, float, str], heuristic: str) -> float:
+def transform_age(value: str, heuristic: str) -> float:
+    if heuristic in ["bg:float", "bg:int"]:
```

Comment:
this is great, but we should probably also cast the input `value` to str where we're calling `transform_age`, e.g. [here](https://github.com/neurobagel/bagel-cli/pull/92/files#diff-b0983ec8b7729431df0779f42e11ff918eacd934e91a6df1fde5a92c28ba1698R155)

Reply:
good catch